src/Employer/Employer.Web/Mappings/VacancyDisplayViewModelMapper.cs

```diff
@@ -42,7 +42,7 @@ public async Task MapFromVacancyAsync(DisplayVacancyViewModel vm, Vacancy vacanc
             vm.ApplicationInstructions = vacancy.ApplicationInstructions;
             vm.ApplicationUrl = vacancy.ApplicationUrl;
             vm.CanDelete = vacancy.CanDelete;
-            vm.CanSubmit = vacancy.CanSubmit;
+            vm.CanSubmit = vacancy.CanSubmit && vacancy.Status != VacancyStatus.Review;
```

Comment:
should this check be moved to the vacancy domain object if it is a behaviour of all vacancies? Then the mapping to the view model would be straight `vm.CanSubmit = vacancy.CanSubmit`?

Reply:
introduced this check to get the  correct link displayed on the page either Submit or Approve Job Advert... 
I got  viewmodel  mapper CanReview does the job. 
i have removed this check. nice one to get this sorted.